src/ManifestStrategy/ConfigureEnvManifestStrategy.php

```diff
@@ -14,6 +14,16 @@
 
 class ConfigureEnvManifestStrategy extends AbstractManifestStrategy
 {
+    /**
+     * @var string
+     */
+    protected const CONFIG_FILE_EXIST_ERROR = 'File `%s` does not exist. Please check file path or customize using `IntegratorConfig::getConfigPath()`.';
+
+    /**
+     * @var string
+     */
+    protected const CONFIG_FILE_SUCCESSFULLY_UPDATED = 'Global config `%s` was added with to %s value `%s`';
```

Comment:
Moved to `$inputOutput`, it only habbit, in fes stream we always moved it to constant because it allows easy to find message by double shift in phpstorm.